src/pages/academy/blog/index.tsx

```diff
@@ -8,7 +8,7 @@ import { AllArticlesQuery } from 'types/graphql.types'
 import Layout from 'components/layout/layout'
 import { SEO, Container, Flex } from 'components/containers'
 import { Header } from 'components/elements'
-import { localize, WithIntl } from 'components/localization'
+import { localize } from 'components/localization'
```

Comment:
For using `localize` function let's check with @swati to see if we need translated `title` and `description` for our SEO tag. If we don't need it we can remove `localize` function from all Academy components.

Reply:
Thank u @mohammad-rework for your revision, I think what you said is out of the scope of this card, I just want to fix ts errors here.
do we need to check it here?

Reply:
If we are only focusing on TS errors in here, we can skip it.